content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -9,7 +9,7 @@ weight: 370
 <!-- overview -->
 This page shows how to configure a Key Management Service (KMS) provider and plugin to enable secret data encryption.
 In Kubernetes {{< skew currentVersion >}} there are two versions of KMS at-rest encryption.
-You should use KMS v2 if feasible because KMS v1 is deprecated (since Kubernetes v1.28).
+You should use KMS v2 if feasible because KMS v1 is deprecated (since Kubernetes v1.28) and disabled by default (since Kubernetes v1.29).
 However, you should also read and observe the **Caution** notices in this page that highlight specific
 cases when you must not use KMS v2.  KMS v2 offers significantly better performance characteristics than KMS v1.
 
```

Comment:
```suggestion

{{< caution >}}
This documentation is for the generally available implementations of KMS v1 and v2.
If you are using any control plane components older than Kubernetes v1.29, please check
the equivalent page in the documentation for the version of Kubernetes that your cluster
is running.
{{< /caution >}}

```
?

Reply:
@sftim I have removed all the references to old Kubernetes version from the doc, so wasn't sure if this is required. LMK what you think!

> This documentation is for the generally available implementations of KMS v1 and v2.

KMS v1 was in beta and has been deprecated. Only KMS v2 is GA.

Reply:
Try this, then:
```suggestion

{{< caution >}}
This documentation is for the generally available implementation of KMS v2 (and for the
deprecated version 1 implementation).
If you are using any control plane components older than Kubernetes v1.29, please check
the equivalent page in the documentation for the version of Kubernetes that your cluster
is running. Earlier releases on Kubernetes had different behavior that may be relevant
for information security.
{{< /caution >}}

```
?

Reply:
added it!